.gitignore

```diff
@@ -158,5 +158,7 @@ GoogleCloudKey.json
 
 # Prod variables
 terraform/.env.prod
+.terraform/
```

Comment:
Should terraform folder be ignored?

Reply:
It should, as terraform executes in GitHub Actions VM

Reply:
Ah yes. I did not see the . before all so I thought it was the whole terraform/ folder 😅